test/linter/test.sh

```diff
@@ -45,6 +45,11 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+helm lint charts/opa-kube-mgmt --strict --set authz.enabled=true --set authz.mgmtToken.secretName=secretName --set authz.mgmtToken.secretKey=secretKey
```

Comment:
Is possible, add also _negative_ cases, i.e. that intentionally will fail template rendering.

Reply:
I followed the pattern from the other lint tests in the file above. There are many ways lint could break when making changes to the chart template itself during development. It is not straightforward to break it with just passing values. Regardless, I think the most important thing is to check that it passes.